src/index.html

```diff
@@ -1,17 +1,184 @@
+        <fieldset class="fieldset">
+          <legend>Personal information</legend>
+          <div class="form-field">
+            <label for="surname">
+              Surname:
+            </label>
+            <input type="text" id="surname">
+          </div>
+          <div class="form-field">
+            <label for="name">
+              Name:
+            </label>
+            <input type="text" id="name">
+          </div>
+          <div class="form-field">
+            <label for="age">
+              How old are You?
+            </label>
+            <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+            >
+          </div>
+          <div class="form-field">
+            <label for="dateofbirth">
+              Full date of birth:
+            </label>
+            <input 
+              type="date" 
+              name="dateof" 
+              id="dateofbirth"
+            >
+          </div>
+          <div>
+            <label for="agreewith1">
+              I accept the term of the agreement
+            </label>
+            <input 
+              type="checkbox" 
+              id="agreewith1"
+            >
+          </div>
+        </fieldset>
+  
+        <fieldset class="fieldset">
+          <legend>Registration</legend>
+  
+          <div class="form-field">
+            <label for="email">
+              E-mail:
+            </label>
+            <input 
+              type="email"
+              id="email"
+              placeholder="email@example.com"
+              required
+            >
+          </div>
+          <div>
+            <label for="Password">
+              Password:
+            </label>
+            <input
+              type="password" 
+              id="Password"
+              required
+              minlength="8"
+              maxlength="15"
+            >
+          </div>
+        </fieldset>
+  
+        <fieldset class="fieldset">
+          <legend>An interesting fact about you!</legend>
+
+          <div class="form-field">
+            Do you love cats?
+            <label>
+              <input
+                type="radio"
+                name="lovesCats"
+                id="catsYes"
+              >
+              Yes
+            </label>
+            <label>
+              <input
+                type="radio"
+                name="lovesCats"
+                id="catsNo"
+              >
+              No
+            </label>
+          </div>
+          <div class="form-field">
+            <label for="colorChoose">
+              What is your favorite color?
+            </label>
+            <input type="color" id="colorChoose">
+          </div>
+          <div class="form-field">
+            <label for="time">
+              What time do you go to bed? 
+            </label>
+            <input
+              type="time" 
+              id="time"
+            >
```

Comment:
```suggestion
            <input type="time"  id="time">
```
2 attributes go in 1 line